subsys/bluetooth/controller/Kconfig

```diff
@@ -51,9 +51,11 @@ config BT_CTLR_DTM_HCI_SUPPORT
 config BT_CTLR_SMI_SUPPORT
 	bool
 
+config BT_CTLR_CONN_RSSI_SUPPORT
+	bool
+
```

Comment:
```suggestion
config BT_CTLR_CONN_RSSI_SUPPORT
	select BT_CTLR_CONN_RSSI if BT_CONN && BT_HCI_RAW
	bool

```

Reply:
I disagree for this one. None of the other _SUPPORT configurations are used to enable features. I would prefer to keep this one as is for consistency.

Reply:
An example of hidden Kconfig that selects other Kconfig: https://github.com/zephyrproject-rtos/zephyr/blob/master/arch/arm/core/aarch32/Kconfig#L6

It is not an exception.

Reply:
ok on second thought, I like the `default y if BT_HCI_RAW` instead as it will allow users to disable RSSI feature to reduce power consumption.

Reply:
I see that I was not clear enough. I was considering the fact that the Bluetooth Controller _SUPPORT features are not used to enable features.